README.md

```diff
@@ -4,12 +4,16 @@
 
 ### Installation
 
-Run the script `install.sh` with **sudo privileges** to install `betty-style` and `betty-code` on your computer, along with the  following manuals:
+First set this gitub repository on your local machine with:  
+
+`git remote add origin https://github.com/holbertonschool/Betty.git`.  Then run `git fetch origin`, then `git merge origin/master`.  Then, run the script `install.sh` with **sudo privileges** to install `betty-style` and `betty-code` on your computer.  If using vagrant, run `$ sudo ./install.sh`.  Now, you will have the Betty installation along with the  following manuals:
```

Comment:
If using vagrant, run `$ sudo ./install.sh` -> That's what the previous sentence says with "with **sudo privileges**" :)